flang/test/Lower/OpenMP/use-device-ptr-to-use-device-addr.f90

```diff
@@ -2,72 +2,72 @@
 ! RUN: %flang_fc1 -emit-hlfir -fopenmp -fopenmp-version=50 %s -o - | FileCheck %s
 ! RUN: bbc -emit-hlfir -fopenmp -fopenmp-version=50 %s -o - | FileCheck %s
 
-! This tests primary goal is to check the promotion of 
-! non-CPTR arguments from use_device_ptr to 
-! use_device_addr works, without breaking any 
-! functionality 
+! This tests primary goal is to check the promotion of
+! non-CPTR arguments from use_device_ptr to
+! use_device_addr works, without breaking any
+! functionality
```

Comment:
Nit: Lines in this comment seems to have been split very short. Since you're already making changes to it, might as well fix that too.
```suggestion
! This tests primary goal is to check the promotion of non-CPTR arguments from
! use_device_ptr to use_device_addr works, without breaking any functionality.
```